src/components/calcite-date-picker/utils.spec.ts

```diff
@@ -28,6 +28,13 @@ describe("utils", () => {
       expect(requestCache).toHaveProperty("en");
     });
 
+    it("falls to lang code locale if regional code is not found", async () => {
+      const locale = "pt-BR";
```

Comment:
Heads up, we may add `pt-BR` (depending on https://github.com/Esri/calcite-components/issues/3925#issuecomment-1018026093), so could you use another region to prevent this test from breaking? Maybe `pt-UnsupportedRegion` to make the test clearer and less brittle?